frontend/packages/ui/src/ProgressBar.tsx

```diff
@@ -88,7 +88,7 @@ export const ProgressBar = ({
       return (
         <LegendPiece key={stats.name}>
           <LegendCircle color={ColorChart[stats.name]} />
-          {stats.name}:{' '}
+          {TestNamings(stats.name)}:{' '}
```

Comment:
I don't think that tooltips are coverd here, only legend and MatrixChart column are coverd in this PR.